_includes/allsids.txt

```diff
@@ -8,9 +8,15 @@
       {% elsif post.vote-status == 'deferred' %}
         <span class="label warning">Deferred</span>
       {% elsif post.vote-status == 'postponed' %}
-        <span class="label warning">Postponed</span>
+        <span class="label light-warning">Postponed</span>
+      {% elsif post.vote-status == 'under review' %}
+        <span class="label warning">Under review</span>
+      {% elsif post.vote-status == 'numbered' %}
+        <span class="label notice">Numbered</span>
       {% elsif post.vote-status == 'not accepted' %}
-        <span class="label danger">Not Accepted</span>
+        <span class="label danger">Rejected</span>
```

Comment:
Why was it changed back to Rejected? I was surprised by the language. Just curious.